velox/exec/HashTable.cpp

```diff
@@ -848,13 +850,13 @@ namespace {
 // Returns an index into 'buildPartitionBounds_' given an index into tags of the
 // HashTable.
 int32_t
-findPartition(int32_t index, const int32_t* bounds, int32_t numPartitions) {
+findPartition(int64_t index, const int64_t* bounds, int32_t numPartitions) {
```

Comment:
There may be some performance regression here, since with int32 we can check 8 partitions at a time, and with int64 we can only check 4.  What is the number of partitions in your experiment?

Reply:
@Yuhta Already fixed. Appreciate it! Let me know if there is anything else I should do.